src/EnergyPlus/GroundTemperatureModeling/KusudaAchenbachGroundTemperatureModel.cc

```diff
@@ -167,7 +161,7 @@ std::shared_ptr<KusudaGroundTempsModel> KusudaGroundTempsModel::KusudaGTMFactory
         }
     }
 
-    if (found && !ErrorsFound) {
+    if (found) {
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/KusudaAchenbachGroundTemperatureModel.cc:170]:(style),[knownConditionTrueFalse],Condition '!ErrorsFound' is always true